### ISSUE TITLE
$view->js('click')->hide() not working

### ISSUE THREAD

**Issue by romaninsh** [atk4/ui#47]
This works:

```
$view->on('click')->hide();
```

but this does not

```
$view->js('click')->hide();
```

**Comment by DarkSide666** [comment]
This works perfectly in grid PR branch:
```
$b = $layout->add(new Button(['id'=>'b2']))->set('Hide on click Button');
$b->js('click')->hide();
```

**Comment by romaninsh** [comment]
what about:

```
$b->js('click', $b2->hide());
```

**Comment by DarkSide666** [comment]
That doesn't work because of issue described here: https://github.com/atk4/ui/issues/48

**Comment by romaninsh** [comment]
Fixed in 456a8f3
